var/spack/repos/builtin/packages/pcre2/package.py

```diff
@@ -27,6 +30,9 @@ def configure_args(self):
             args.append('--enable-pcre2-16')
             args.append('--enable-pcre2-32')
 
+        if '+jit' in self.spec:
+            args.append('--enable-jit')
```

Comment:
Is it possible to explicitly disable this in an else clause?

Reply:
@adamjstewart , by default `jit` is disabled here is the part from configure file in tar source.
```
# Handle --enable-jit (disabled by default)
# Check whether --enable-jit was given.
if test "${enable_jit+set}" = set; then :
  enableval=$enable_jit;
else
  enable_jit=no
fi
```